charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.6.5
```

Comment:
```suggestion
## 2.7.0
```

As the new version of the Helm chart requires a newer version of the agent and cluster agent, we should bump the minor version of the Helm chart.